media/css/protocol/basic-article.scss

```diff
@@ -41,6 +41,15 @@ $image-path: '/media/protocol/img';
     }
 }
 
+.non-standout-quote {
```

Comment:
`blockquote.t-small`?

Reply:
Same as above - as long as the copy looks roughly the same happy to defer.